core/meta/src/TCling.cxx

```diff
@@ -213,33 +213,59 @@ using namespace clang;
 using namespace ROOT;
 
 namespace {
-   static const std::string gInterpreterClassDef = "#undef ClassDef\n"
-                "#define ClassDef(name, id) \\\n"
-                "_ClassDefInterp_(name,id,virtual,) \\\n"
-                "static int DeclFileLine() { return __LINE__; } \n"
-                "#undef ClassDefNV\n"
-                "#define ClassDefNV(name, id) \\\n"
-                "_ClassDefInterp_(name,id,,) \\\n"
-                "static int DeclFileLine() { return __LINE__; }\n"
-                "#undef ClassDefOverride\n"
-                "#define ClassDefOverride(name, id) \\\n"
-                "_ClassDefInterp_(name,id,,override) \\\n"
-                "static int DeclFileLine() { return __LINE__; }\n";
-   static const std::string gNonInterpreterClassDef = "#define __ROOTCLING__ 1\n"
-               "#undef ClassDef\n"
-               "#define ClassDef(name,id) \\\n"
-               "_ClassDef_(name,id,virtual,) \\\n"
-               "static int DeclFileLine() { return __LINE__; }\n"
-               "#undef ClassDefNV\n"
-               "#define ClassDefNV(name, id)\\\n"
-               "_ClassDef_(name,id,,)\\\n"
-               "static int DeclFileLine() { return __LINE__; }\n"
-               "#undef ClassDefOverride\n"
-               "#define ClassDefOverride(name, id)\\\n"
-               "_ClassDef_(name,id,,override)\\\n"
-               "static int DeclFileLine() { return __LINE__; }\n";
-}
+  static const std::string gInterpreterClassDef = R"ICF(
+#undef ClassDef
+#define ClassDef(name, id) \
+_ClassDefInterp_(name,id,virtual,) \
+static int DeclFileLine() { return __LINE__; }
+#undef ClassDefNV
+#define ClassDefNV(name, id) \
+_ClassDefInterp_(name,id,,) \
+static int DeclFileLine() { return __LINE__; }
+#undef ClassDefOverride
+#define ClassDefOverride(name, id) \
+_ClassDefInterp_(name,id,,override) \
+static int DeclFileLine() { return __LINE__; }
+)ICF";
+
+  static const std::string gNonInterpreterClassDef = R"ICF(
+#define __ROOTCLING__ 1
+#undef ClassDef
+#define ClassDef(name,id) \
+_ClassDef_(name,id,virtual,) \
+static int DeclFileLine() { return __LINE__; }
+#undef ClassDefNV
+#define ClassDefNV(name, id)\
+_ClassDef_(name,id,,)\
+static int DeclFileLine() { return __LINE__; }
+#undef ClassDefOverride
+#define ClassDefOverride(name, id)\
+_ClassDef_(name,id,,override)\
+static int DeclFileLine() { return __LINE__; }
+)ICF";
+
+// The macros below use ::Error, so let's ensure it is included
+  static const std::string TErrorClassDef = R"ICF(
```

Comment:
The var name focuses too much on TError - but that's really just a minor implementation detail. I'd prefer to call this variable `gClassDefInterpMacro`